roles/pgbench/tasks/run_workload.yml

```diff
@@ -31,5 +70,5 @@
       - app = pgbench-client
   register: pgbench_pods
   until: "'Succeeded' in (pgbench_pods | json_query('resources[].status.phase'))"
```

Comment:
I noted this in chat, but this will *stop* the first pod that completes. I have a fix, and we don't need to block for this. Just noting.